src/modules/Client/Service.php

```diff
@@ -100,7 +100,7 @@ public static function onAfterClientSignUp(\Box_Event $event)
             $email = [];
             $email['to_client'] = $params['id'];
             $email['code'] = 'mod_client_signup';
-            $email['password'] = $params['password'];
+            $email['password'] = 'The password you choose when creating your account.';
```

Comment:
Would it be necessary to email this, then? Why not remove it altogether?
I also think the verb would be better in its past form ("chose"), not sure, what do you think?

Reply:
Whoops, I didn't see @BelleNottelling's response above where the same thing was mentioned.

Reply:
For backwards compatibility with existing email templates, the variable should probably still be passed.
Ultimately, the email template determines if it's sent at all

Reply:
Also admin can create the user in that case the password is required or tell the user set their unique password via a link

Reply:
If an Admin is creating a user the password shouldn't really be sent via email. Maybe a reset password link could be sent? Or if a password is sent it should require the user to change it on login.

Reply:
M365 does gives the option to send it or not.  

Ideally we should send a "temp" password but that would require more changes in the software..

Reply:
We can worry about these other issues at a later date. For now, This is a good step in the right direction